packages/mdc-slider/index.js

```diff
@@ -116,6 +119,28 @@ export class MDCSlider extends MDCComponent {
       setTrackStyleProperty: (propertyName, value) => {
         this.track_.style.setProperty(propertyName, value);
       },
+      setMarkerValue: (value) => {
+        this.pinValueMarker_.innerText = value;
+      },
+      appendTrackMarkers: (numMarkers) => {
```

Comment:
The reason why I have it as an adapter method is because mdc-slider need to able to dynamically change the markers on track when users calls `setMax`, `setMin`, etc...

Also this cannot be done in `initialize` because `initialize` run before `foundation.init`, but we are actually doing this in `initialSyncWithDOM`.